components/SearchBar/index.js

```diff
@@ -86,19 +90,21 @@ function SearchBar({ search, onSearch }) {
           </Styled.FilterContainer>
           <Styled.FilterContainer>
             <Styled.FilterItem>
-              <label>Date Start</label>
+              <label for="date_start">Date Start</label>
               <Styled.SearchInput
                 type="date"
+                id="date_start"
                 value={search.dateFrom}
                 onChange={(event) =>
                   onSearch({ ...search, dateFrom: event.target.value })
                 }
               />
             </Styled.FilterItem>
             <Styled.FilterItem>
-              <label>Date End</label>
+              <label for="date_end">Date End</label>
```

Comment:
[We need to use `htmlFor` in React](https://legacy.reactjs.org/docs/dom-elements.html#htmlfor).